src/types/profile-derived.js

```diff
@@ -256,7 +256,8 @@ export type OriginsTimeline = Array<
  */
 export type ActiveTabMainTrack = {|
   type: 'tab',
-  threadIndex: ThreadIndex,
+  mainThreadIndex: ThreadIndex,
```

Comment:
I still keep a `mainThreadIndex` because most of the places, code depends on having a single thread index (like url state, transforms etc.)